RecoLocalTracker/SiPixelClusterizer/plugins/BuildFile.xml

```diff
@@ -1,20 +1,35 @@
 <use name="boost_serialization"/>
-<use name="CUDADataFormats/SiPixelCluster"/>
-<use name="CUDADataFormats/SiPixelDigi"/>
-<use name="CalibTracker/SiPixelESProducers"/>
 <use name="DataFormats/Common"/>
 <use name="DataFormats/SiPixelCluster"/>
+<use name="DataFormats/SiPixelClusterSoA"/>
+<use name="DataFormats/SiPixelDigiSoA"/>
 <use name="EventFilter/SiPixelRawToDigi"/>
 <use name="FWCore/ParameterSet"/>
 <use name="FWCore/Utilities"/>
-<use name="HeterogeneousCore/CUDACore"/>
 <use name="RecoTracker/Record"/>
+<use name="CalibTracker/SiPixelESProducers"/>
+<use name="CondFormats/SiPixelObjects"/>
+
 <iftool name="cuda-gcc-support">
   <use name="cuda"/>
+  <use name="HeterogeneousCore/CUDACore"/>
+  <use name="CUDADataFormats/SiPixelCluster"/>
+  <use name="CUDADataFormats/SiPixelDigi"/>
```

Comment:
Maybe these dependencies (along the `cuda`) would be better to be moved inside the definition of `RecoLocalTrackerSiPixelClusterizerPlugins`, so that they would not affect `RecoLocalTrackerSiPixelClusterizerPluginsPortable`?

Reply:
I think this is now done.